tests/ignite/contrib/metrics/test_precision_recall_curve.py

```diff
@@ -124,3 +127,95 @@ def test_check_compute_fn():
 
     em = PrecisionRecallCurve(check_compute_fn=False)
     em.update(output)
+
+
+def _test_distrib_binary_input(device):
```

Comment:
This method does not test anything. As `_test` method is never called, the executed code is empty.

Please have a look here :
https://github.com/pytorch/ignite/blob/3d9ba50bba42438a63183b3db0d4cc3a5e9b01ac/tests/ignite/contrib/metrics/regression/test_median_absolute_error.py#L152

The code would be very similar.